python/utils/LinkedLibraryHolder.cpp

```diff
@@ -30,6 +30,30 @@ constexpr static const char NVQIR_SIMULATION_BACKEND[] =
     "NVQIR_SIMULATION_BACKEND=";
 constexpr static const char TARGET_DESCRIPTION[] = "TARGET_DESCRIPTION=";
 
+int countGPUs() {
```

Comment:
any reason we shouldn't be using the CUDA runtime library for this? maybe we don't want that dependency here?

Reply:
It may not be available at the point of this check.

Reply:
Do you mean it can't be loaded prior to calling this function?

Reply:
Yes, at least I couldn't invoke it at this point.